crates/tabby-common/src/config.rs

```diff
@@ -152,6 +153,16 @@ pub enum ModelConfig {
     Local(LocalModelConfig),
 }
 
+impl Default for ModelConfig {
+    fn default() -> Self {
+        ModelConfig::Local(LocalModelConfig {
+            model_id: "Nomic-Embed-Text".to_string(),
+            parallelism: 1,
+            num_gpu_layers: 9999,
+        })
+    }
+}
+
```

Comment:
i don't think it's a good pratcie to make it default for `ModelConfig` - as it's also reused for chat / completion.

doing this with something like `serde(default="default_embedding_config")` should be better?